src/data/modules_i.json

```diff
@@ -158,13 +169,16 @@
   "PRAXIS_3/6": "Projektmodul",
   "PRAXIS_VT": "Erweiterungsmodul",
   "PRAXIS_VTZ": "Projektmodul",
-  "PREN1": "Projektmodul",
+  "PREN1": "Erweiterungsmodul",
```

Comment:
The parser still works fine. PREN1 is now actually listed in the section "Erweiterungsmodule" for whatever reason.

Reply:
In that case the required minimum credits for "Projektmodule" and "Erweiterungsmodule" have changed. We need to check that to make sure the calculations are still correct.

Are there still students studying right now where PREN is counted as "Projektmodul"? Because then we have the salad! 😅

Reply:
There is a new module called iPREN1 and iPREN2 which are more focused on IT according to email. We still have the problem, that we can't distinguish when the module was taken and which type it had back then.

Reply:
Then we might need an option to manually change the type of a module.